### ISSUE TITLE
Incompatibility in Schemas When Migrating from Cassandra to Scylla 

### ISSUE THREAD

**Issue by guy9** [scylladb/scylla#9859]
Scylla version: 4.5
Cluster size: 3
Cassandra version: 4.0
Platform: Docker

When trying to export a Schema from a Cassandra cluster and importing it to a Scylla cluster the import fails because of some incompatibilities. 
For each of the items below, I think we should choose one of the following:
1. Continue to give an error when trying to import. In that case, we should consider giving a message with more details and updating our Docs to reflect this.
2. Ignore the line, for example, ignore additional_write_policy = '99p' as Scylla doesn't support this. (@avikivity commented: "We can't ignore unknown properties, since we don't know if they are needed for correctness")
3. Fix the line by converting it, for example "cdc = false" will turn into cdc = {'enabled': 'false'}

### Incompatibilities
1. cdc = false in the Cassandra schema, causes "ServerError: Cannot create cdc info from string". Scylla expects a dictionary such as cdc = {'enabled': 'false'}
2. additional_write_policy = '99p' Scylla doesn't support this and fails with "SyntaxException: Unknown property 'additional_write_policy'" (@avikivity commented: We don't support additional_write_policy , but it seems to be an alias for speculative_retry that we do support)
3. extensions = {} fails with SyntaxException: Unknown property 'extensions'
4. read_repair = 'BLOCKING' fails with SyntaxException: Unknown property 'read_repair'
5. ConfigurationException: Missing sub-option 'sstable_compression' for the 'compression' option and
6. compression = {'chunk_length_in_kb': '16', 'class': 'org.apache.cassandra.io.compress.LZ4Compressor'} causes ConfigurationException: Unknown compression option 'class'.
The solution for 5 and 6 above is to replace under "compression" 'class': 'org.apache.cassandra.io.compress.LZ4Compressor' with 'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'

Attached is a schema file for reference. file:///home/guy/Downloads/orig_schema.cql

To recreate: 
1. Start a 3 node Cassandra cluster
2. Start a 3 node Scylla cluster
3. Export schema from Apache Cassandra: cqlsh [IP] "-e DESC SCHEMA" > orig_schema.cql
4. Import schema to Scylla: cqlsh [IP] --file 'adjusted_schema.cql'

Also related to [this](https://github.com/scylladb/scylla-doc-issues/issues/828) Documentation issue. 

CC @tzach and @avikivity

**Comment by guy9** [comment]
Additionally:
7. speculative_retry = '99p' fails on import with "ServerError: sstring out of range"
Changing it to: "speculative_retry = '99.0PERCENTILE';"
resolves it.

**Comment by tzach** [comment]
@guy9 should we close this issue?

**Comment by guy9** [comment]
I don't think we shouldn't close it. Instead, for each one of the 7 incompatibilities mentioned we should choose one of the following:

1. Continue to give an error when trying to import. In that case, we should consider giving a message with more details and updating our Docs to reflect this.
2. Ignore the line, for example, ignore additional_write_policy = '99p' as Scylla doesn't support this. (@avikivity commented: "We can't ignore unknown properties, since we don't know if they are needed for correctness")
3. Fix the line by converting it, for example "cdc = false" will turn into cdc = {'enabled': 'false'}

Additionally, I would suggest further testing for incompatibilities in the migration process. Do we have an automated test for this?
I doubt I found all issues as I just tested one schema.

**Comment by nyh** [comment]
The missing support for "class" in compression options (incompatibility 6 above) is issue #2061.
